firmware/application/apps/ui_freqman.cpp

```diff
@@ -163,9 +177,13 @@ FrequencySaveView::FrequencySaveView(
 
     bind(field_description, entry_.description, nav);
 
-    button_save.on_select = [this, &nav](Button&) {
-        db_.insert_entry(db_.entry_count(), entry_);
-        nav_.pop();
+    button_save.on_select = [this, &nav](Button&) {  // TODO: don't list system category here
+        if (current_is_system_item) {
+            nav.display_modal("Forbid", "Can't save to system Category\nPlease save to \na user category.");
```

Comment:
Why is the word "category" capitalized on the first line (and then not capitalized on the next line)?